### PR TITLE
Improve button groupings (first tranche)

### PR DESCRIPTION
**Pull request by MylesJarvis**
### Trello card

https://trello.com/c/QHKZbml7

### Context

- We have some issues relating to how our buttons are grouped, we too often rely on putting them on separate lines, which does not align with the Design System.

### Changes proposed in this pull request (changes in comments below)

- Improve button grouping on the confirm attendance page
- Add a back link on the 'toggle profile on/off' page and group this with existing buttons
- Improve button grouping on our onboarding flow
- Improve button grouping on our edit profile flow
- Improve button grouping on our 'add dates' page

### Guidance to review

Attendance page (you will need to login as a school with attendances to confirm (Holland Park) and one without (Outwood Academy Acklam)):
- https://get-school-experience-review-pr-4.test.teacherservices.cloud/schools/confirm_attendance

Toggle page on/off:
- https://get-school-experience-review-pr-4.test.teacherservices.cloud/schools/toggle_enabled/edit

Edit profile page (use any onboarded school)
- https://get-school-experience-review-pr-4.test.teacherservices.cloud/schools/on_boarding/profile#admin-contact-details-tab

Add dates (use any school with 'fixed dates'):
- https://get-school-experience-review-pr-4.test.teacherservices.cloud/schools/placement_dates

Onboarding flow (use any non-onboarded school and complete their profile details)
- https://get-school-experience-review-pr-4.test.teacherservices.cloud/schools/on_boarding/profile?

For the moment the main focus can be grouping - I can do separate PRs for changing buttons -> links if needed. For example, the return to dashboard buttons.

### CODE REVIEW

**Comment by MylesJarvis** [comment]
Splitting each change into comments to make it easier to review:

**Before - turn profile on/off**

image


**After - turn profile on/off**

image

**Comment by MylesJarvis** [comment]
**Before**
image

image


**After**
image

image

**Comment by MylesJarvis** [comment]
**Before**
image
**After**
image

**Comment by MylesJarvis** [comment]
**Before**
image

**After**
image

**Comment by MylesJarvis** [comment]
**Before**

image


**After**

image

File: app/views/schools/dashboards/_no_placement_dates_warning.html.erb [hunk 1]
```diff
@@ -1,8 +1,8 @@
-<div class="govuk-warning-text govuk-!-padding-top-7">
+<div class="govuk-warning-text">
   <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
   <strong class="govuk-warning-text__text">
     <span class="govuk-visually-hidden">Warning</span>
     You haven't entered any dates, your school will not appear in candidate
```
Comment (gemmadallmandfe) [inline app/views/schools/dashboards/_no_placement_dates_warning.html.erb:5]:
```suggestion
    You haven't entered any dates so your school will not appear in candidate
```

**Review by ekumachidi — APPROVED** [review]

**Pull request merged**